experimental/FTheoryTools/src/G4Fluxes/constructors.jl

```diff
@@ -153,6 +155,17 @@ function Base.show(io::IO, g4::G4Flux)
     push!(properties_string, "  - Verticality checks: not executed")
   end
 
+  # Check for non-Abelian gauge group breaking
+  if has_attribute(g4, :breaks_non_abelian_gauge_group)
+    if breaks_non_abelian_gauge_group(g4)
+      push!(properties_string, "  - Non-Abelian gauge group: broken")
```

Comment:
Just out of curiosity, why `Non-Abelian` and not the more usual `non-abelian`?

Reply:
As a humble PhD student, @mohamed-barakat once showed me his (at the time) writing style for proposals, in which the capitalized the name of Mathematicians to honor their contributions. In the case at hand, Abel. I believe this style stayed with me since, although I am sure to have been inconsistent.

On second look, I favor `non-abelian` here. More consistent. Will update this shortly.

Reply:
Indeed I just found this on [Wikipedia](https://en.wikipedia.org/wiki/Abelian_group#A_note_on_typography):
> Among mathematical adjectives derived from the proper name of a mathematician, the word "abelian" is rare in that it is often spelled with a lowercase a, rather than an uppercase A, the lack of capitalization being a tacit acknowledgment not only of the degree to which Abel's name has been institutionalized but also of how ubiquitous in modern mathematics are the concepts introduced by him